test/test_bwproject.py

```diff
@@ -0,0 +1,69 @@
+import unittest
+import responses
+import os
+import tempfile
+
+from bwproject import BWProject
+
+
+class TestBWProjectUsernameCaseSensitivity(unittest.TestCase):
```

Comment:
This only tests the happy path. It would be good to test that this fails for different usernames. You can use assertRaises as a context manager to check that the exception is thrown.